homeassistant/components/forked_daapd/media_player.py

```diff
@@ -421,10 +421,12 @@ async def async_turn_on(self) -> None:
             futures = []
```

Comment:
I think you can improve the type hint on `futures`:
```suggestion
            futures: list[asyncio.Task[int]] = []
```